src/Stripe.net/Services/Coupons/StripeCouponCreateOptions.cs

```diff
@@ -17,7 +17,7 @@ public class StripeCouponCreateOptions : StripeBaseOptions, ISupportMetadata
         public string Currency { get; set; }
 
         [JsonProperty("percent_off")]
-        public int? PercentOff { get; set; }
+        public decimal? PercentOff { get; set; }
```

Comment:
This was always a decimal and the type was just incorrect in the .NET library, correct?

Reply:
No it was always an integer until API version [2018-07-27](https://stripe.com/docs/upgrades#2018-07-27)